tools/generator/ci.yml

```diff
@@ -25,3 +25,4 @@ stages:
   - template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
     parameters:
       ServiceDirectory: '../tools/generator'
+      RunChangelogCheck: false
```

Comment:
Things under the tools repo do they actually need a full pipeline? Do we release them? I suggest instead of making this specific to verify changelog we might need to have a parameter that indicates that this is not a shipping package and we turn off any thing related to shipping/releasing. There are other things like the release template that might fail for this package also. We could even consider looking at ServiceDirectory and enabling those features only for things under the sdk folder.

Reply:
I've changed the flag to 'NonShipping'. I think splitting the pipeline template for tooling and package shipping is a good idea.